udata/harvest/backends/dcat.py

```diff
@@ -285,7 +285,7 @@ class CswIso19139DcatBackend(DcatBackend):
 
     ISO_SCHEMA = 'http://www.isotc211.org/2005/gmd'
 
-    XSL_URL = "https://raw.githubusercontent.com/SEMICeu/iso-19139-to-dcat-ap/master/iso-19139-to-dcat-ap.xsl"
+    XSL_URL = "https://raw.githubusercontent.com/datagouv/iso-19139-to-dcat-ap/empty_function_to_distribution/iso-19139-to-dcat-ap.xsl"
```

Comment:
:eyes:

Reply:
Reverted in 3403944f7, done in https://github.com/opendatateam/udata/pull/3051